### PR TITLE
add image matching fast model based on lightglue

### PR DESCRIPTION
**Pull request by zhengyi-z**

### CODE REVIEW

File: modelscope/models/cv/image_matching_fast/lightglue_model.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+from modelscope.utils.constant import ModelFile, Tasks
+from .lightglue import LightGlue, SuperPoint, DISK, ALIKED, SIFT
+from .lightglue.utils import rbd, numpy_image_to_torch
+from .config.default import lightglue_default_conf
+
+
+@MODELS.register_module(
+    Tasks.image_matching, module_name=Models.lightglue_image_matching)
+class LightGlueImageMatching(TorchModel):
+
```
Comment (wenmengzhou) [inline modelscope/models/cv/image_matching_fast/lightglue_model.py:24]: add docstr for model and paper link if it has

File: modelscope/pipelines/cv/image_matching_fast_pipeline.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+        Return:
+            A list of result.
+            The list contain the following values:
+
+            - kpts0 -- Matched keypoints in the first image
+            - kpts1 -- Matched keypoints in the second image
+            - confidence -- Confidence of the match
+        """
+        return super().__call__(input, **kwargs)
```
Comment (wenmengzhou) [inline modelscope/pipelines/cv/image_matching_fast_pipeline.py:88]: can be removed

File: modelscope/pipelines/cv/image_matching_fast_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+from modelscope.utils.constant import Tasks
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.image_matching, module_name=Pipelines.image_matching_fast)
+class ImageMatchingFastPipeline(Pipeline):
+
```
Comment (wenmengzhou) [inline modelscope/pipelines/cv/image_matching_fast_pipeline.py:23]: add pipeline doc str, refer to https://github.com/modelscope/modelscope/blob/master/modelscope/pipelines/cv/card_detection_correction_pipeline.py#L34C1-L34C1

File: modelscope/models/cv/image_matching_fast/lightglue_model.py (hunk 1 above)
Comment (zhengyi-z) [reply modelscope/models/cv/image_matching_fast/lightglue_model.py:28]: Done.

File: modelscope/pipelines/cv/image_matching_fast_pipeline.py (hunk 3 above)
Comment (zhengyi-z) [reply modelscope/pipelines/cv/image_matching_fast_pipeline.py:43]: Done.

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**